### ISSUE TITLE
Question about train script

### ISSUE THREAD

**Issue by CinKKKyo** [Pulkit-Khandelwal/medical-mldg-seg#3]
Hi Pulkit, thanks for the work. There are three questions:
Q1: ![image](https://github.com/Pulkit-Khandelwal/medical-mldg-seg/assets/40874032/a81ff136-c7b1-44e1-adc6-1ddeb7f0b4b0)
What does it mean in the "meta_learning_spine.py" file, line 205? 

Q2: Did you use the same number of samples in different domains during the training process? What if the number of samples in different domains is not the same? 

Q3: I saw in your paper, the hyperparameters of meta_learning were both set to 1 (/alpha, //beta, /gamma), whereas the /gamma represents the outer learning rate and it was controlled by the optimizer. But the lr of the optimizer was set to 1e-3, I am not sure what lr of the optimizer(/gamma) you chose in the training process.
